ledger-core/virtual/integration/method_test.go

```diff
@@ -1359,13 +1365,13 @@ func Test_MethodCall_HappyPath(t *testing.T) {
 			content := &payload.VStateReport_ProvidedContentBody{
 				LatestDirtyState: &payload.ObjectState{
 					Reference: reference.Local{},
-					Class:     testwalletProxy.GetClass(),
-					State:     []byte(origObjectMem),
+					Class:     gen.UniqueGlobalRef(),
+					State:     []byte(origDirtyObjectMem),
 				},
 				LatestValidatedState: &payload.ObjectState{
 					Reference: reference.Local{},
-					Class:     testwalletProxy.GetClass(),
-					State:     []byte(origObjectMem),
+					Class:     gen.UniqueGlobalRef(),
```

Comment:
do not generate two different classes, use one, pls.

Reply:
done